src/hotspot/share/gc/z/zHeap.cpp

```diff
@@ -369,11 +388,14 @@ void ZHeap::select_relocation_set() {
       // Register garbage page
       selector.register_garbage_page(page);
 
-      // Reclaim page immediately
-      free_page(page, true /* reclaimed */);
+      // Reclaim garbage pages in bulk
```

Comment:
Maybe adding a comment that this is an optimization?

Reply:
Will fix.

Reply:
Thank you.